integration_test/sql/migration.exs

```diff
@@ -683,4 +741,35 @@ defmodule Ecto.Integration.MigrationTest do
 
     :ok = down(PoolRepo, num, OnDeleteNilifyColumnsMigration, log: false)
   end
+
+  @tag :on_delete_default_all
+  test "default all on_delete constraint", %{migration_number: num} do
+    assert :ok == up(PoolRepo, num, OnDeleteDefaultAllMigration, log: false)
+
+    PoolRepo.insert_all("parent", [%{id: 1, col1: 2, col2: 3}])
+    {id, col1, col2} = {Enum.random(10..1000), Enum.random(10..1000), Enum.random(10..1000)}
```

Comment:
Can we remove the first `insert_all` without affecting the test? The reason I'm asking is because the assertion at the bottom is for the exact values of this record. So it makes it less obvious you are testing the defaults from the migration are being applied.

Another thing is I think it's ok to hard code instead of using random. Since we know the defaults and we just want the new values to be different.

Reply:
sorry re-reading your posts i see that's the point =) ignore me